OneSignalSDK/unittest/src/test/java/com/test/onesignal/GenerateNotificationRunner.java

```diff
@@ -836,7 +842,7 @@ public void shouldRestoreNotifications() {
 
       // Go forward 1 week
       // Note: Does not effect the SQL function strftime
-      advanceSystemTimeBy(604_801);
+      advanceSystemTimeBy(time, 604_801);
```

Comment:
Instead of passing everything into the `advanceSystemTimeBy` maybe we should just call a method inside of `time`?

**Example:**
`time.advanceSystemTimeBy(604_801)`?

Reply:
:not_bad: 👍